docs/python/tutorial-django.md

```diff
@@ -107,8 +116,8 @@ To create a minimal Django app, then, it's necessary to first create the Django
 
     System check identified no issues (0 silenced).
 
-    September 05, 2018 - 14:33:31
-    Django version 2.1.1, using settings 'web_project.settings'
+    January 15, 2021 - 14:33:31
```

Comment:
It will actually raise a set of warnings because `manage.py migrate` has not been run yet. This should really be added as an extra step

Reply:
The official Django Tutorial shows it and deals with database migrations as a separate step.